otx/algorithms/action/configs/detection/two_stage_two_backbone/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Initialization of Two Stage Two Backbone structure for Action Detection Task."""
```

Comment:
We usually set the folder name with the name of `model` or `method` like `x3d`.
However, `two_stage_two_backbone` can include many methods. How about changing the name of this folder?

Reply:
Actually, two_stage_two_backbone is like a structure of action detector such as YOLO or Faster-RCNN, and it can take any action classification model such as SlowFast, X3D. What name could be proper for this situation?

Reply:
How about using `faster_rcnn_with_x3d`? For the detection case, although we can use many kinds of backbones for `SSD / ATSS / YOLOX.`, their name aren't `Onestage Detector`.

Reply:
Change to x3d_fast_rcnn followed from object detection(backbone_structure).